controllers/grid/settings/metadata/MetadataGridHandler.inc.php

```diff
@@ -95,7 +95,7 @@ static function getNames() {
 			'languages' => array('name' => __('rt.metadata.dublinCore.language')),
 			'rights' => array('name' => __('rt.metadata.dublinCore.rights')),
 			'source' => array('name' => __('rt.metadata.dublinCore.source')),
-			'subject' => array('name' => __('rt.metadata.dublinCore.subject')),
+			'subjects' => array('name' => __('rt.metadata.dublinCore.subject')),
```

Comment:
I think this will also require changes to locale keys per https://github.com/pkp/pkp-lib/blob/master/classes/submission/PKPSubmissionMetadataFormImplementation.inc.php#L66.